### DISCUSSION TITLE
Unable to add https://charts.promitor.io

### DISCUSSION THREAD

**Discussion by Takshika** [tomkerkhove/promitor#1751] [General] [answered]
I am not able to add promitor repo. It throws errors - "Error: looks like "https://charts.promitor.io/" is not a valid chart repository or cannot be reached: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go value of type repo.IndexFile"

Followed https://artifacthub.io/packages/helm/promitor/promitor-agent-scraper to add promitor repo. Can someone help me understand if I am taking some wrong path and what would be correct path.

```cli
$ helm version
version.BuildInfo{Version:"v3.6.3", GitCommit:"d506314abfb5d21419df8c7e7e68012379db2354", GitTreeState:"dirty", GoVersion:"go1.16.6"}

$ helm repo add promitor https://charts.promitor.io/
helm repo add promitor https://charts.promitor.io/
Error: looks like "https://charts.promitor.io/" is not a valid chart repository or cannot be reached: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go value of type repo.IndexFile
```

**Comment by tomkerkhove** [comment, marked as answer]
Hm odd.

I tried this on both PowerShell & Bash with  the same Helm version and it worked.
Did you try it multiple times?

**Reply by tomkerkhove** [reply]
Also, are you able to browse to https://charts.promitor.io/?

**Reply by Takshika** [reply]
Yes, I tried it multiple times from MacOs terminal. I am not able to browse  https://charts.promitor.io/. IT throws error as "refused to connect".
(Also, I tried adding another chart, that works fine. )

**Reply by Takshika** [reply]
ok i think it was VPN issue. Its working now. Thanks alot:)

**Reply by tomkerkhove** [reply]
Ok, thanks for letting me know and good luck!

Once you are using it in production, feel free to reach out if you want to be a listed end-user

**Reply by Takshika** [reply]
Yup will definitely update on that. Thank you.
